content/protobuf/index.md

```diff
@@ -1,7 +1,120 @@
 ---
-title: Protobuf Overview
-showAside: false
+title: Protobuf
+description: Using Protocol Buffers with IceRPC
 showReadingTime: false
 ---
 
-This is a placeholder for the Protobuf Overview page.
+## About Protocol Buffers
+
+Protocol Buffers or Protobuf for short is a popular [Interface Definition Language] (IDL) and serialization format
+created by Google and available for many programming languages. [gRPC] and several other RPC frameworks use [Protobuf] as
+their IDL.
+
+This chapter describes the IceRPC + Protobuf integration, not Protobuf itself. The IceRPC + Protobuf integration uses
+standard Protobuf files, with the regular Protobuf syntax and semantics.
+
+## The IceRPC + Protobuf integration
+
+The IceRPC + Protobuf integration helps you implement Protobuf services with IceRPC and call such services using (over)
+IceRPC.
+
+In practical terms, the IceRPC + Protobuf integration for C#:
+
+- defines an IceRPC-specific client-side and server-side mapping for Protobuf services to C#
+- provides a `protoc` plugin (`protoc-gen-icerpc-csharp`) that generates code for Protobuf services
+- provides a support assembly used by this generated code, `IceRpc.Protobuf.dll`
+- provides a MSBuild integration that helps you execute `protoc` with the `protoc-gen-icerpc-csharp` plugin in MSBuild
```

Comment:
Because M is pronounced `em`, starting with a vowel sound.
```suggestion
- provides an MSBuild integration that helps you execute `protoc` with the `protoc-gen-icerpc-csharp` plugin in MSBuild
```